### ISSUE TITLE
Conductor: Processing `verification_request` from work queue

### ISSUE THREAD

**Issue by cameel** [golemfactory/concent#208]
This task depends on #207.

### Sequence of operation
Implement the following part of the #193 blueprint:
- Conductor (HTTP) receives a request from nginx-storage (see https://github.com/golemfactory/concent-deployment/issues/89).
- The app creates a new instance of `UploadReport` in the database.
    - If there's a corresponding `UploadRequest`, the `UploadReport` links to it.
- If the `UploadRequest` exists, it's linked with a `VerificationRequest`. The app gets the `VerificationRequest` and checks if all `UploadRequest` instances have reports.
- If all expected files have been uploaded, the app sends `verification_order` task to the work queue.

### `verification_order` task
Define a Celery task called `verification_order` with the following arguments:
- All relevant fields from `VerificationRequest`
- `source_file`: Relative path of the .zip file that contains Blender source files for the render
- `result_file`: Relative path of the .zip file that contains the rendering result received from the provider.

### Setup for local development
See #201 for instructions on how to set up a local message queue for development.

You'll also need to simulate requests from nginx-storage. https://github.com/golemfactory/concent-deployment/issues/89 shows how requests from it will look like.

**Comment by cameel** [comment]
Update: task separation. (see description)

**Comment by cameel** [comment]
Update: ignore my previous update :) I just realized that the name of this issue was wrong. It's about processing upload reports, not `verification_request`.

**Comment by cameel** [comment]
Update: tasks and models have changed in the blueprint:
- `verification_order` renamed to `blender_verification_order`.
- Many fields have been removed or changed in `verification_order`.
- `UploadRequest` model has been removed. Now `VerificationRequest` specifies the two files that must be uploaded.

**Comment by cameel** [comment]
Update: endpoint renamed to `/report-upload/`. But please don't reopen it. There's a separate issue for fixing this: #327
